config/setup/common/data/system_ingest_config.dat

```diff
@@ -46,6 +46,8 @@ pipeline:
           processor:
             - http:
                 max_sending_qps: 100
+                compress: true
```

Comment:
I prefer setting compress to false by default to prioritize lower resource consumption. Also, we shouldn’t ignore HTTP 413 status codes by default, as doing so could mask underlying issues and lead to confusion.